src/osvr/Server/ConfigFilePaths.cpp

```diff
@@ -77,20 +77,26 @@ namespace osvr {
             configDir /= fs::path("OSVR") / configSubpath;
 #endif
 
-            return configDir.string();
+#if defined(OSVR_ANDROID)
+            configDir = fs::path("/sdcard/osvr");
+#endif
+            auto ret = configDir.string();
```

Comment:
No particular reason. I was diagnosing a stdlib error in this code and tried splitting it up to see if it was happening in string() and left the change in. OK to leave in, or should I revert it?